proposals/geolocation.md

```diff
@@ -0,0 +1,404 @@
+This should be relative to the "natural" surface before man-made excavations.
+For example, a location in a basement should have a negative depth.
+
+If an elevation is also given, then the depth is relative to that elevation
+instead of the surface.
+
+#### Orientation
+Both 
+* 2D Heading (azimuth) - angle from north
+* Incline (elevation) - deviation from vertical
+* Tilt (rotation around heading)
+Issues:
+* Singularies e.g. at the pole
+* Interpolation
+* Alignment with Web API
+* Represention: Euler angles, Quaternions, Vector/Angle
+* Default values
+
+#### Velocity
+While velocity can be estimated from two successive positions,
+if known directly it can be reported.
+This is a 3D vector value defined in the coordinate system
+given by the orientation.
+
+#### Speed
+An alternative simple specification of velocity when the 
+velocity vector is the same as the orientation.  In this case
+only a scalar providing the length of the velocity vector is given.
+
+#### Acceleration
+While this can be estimated from two successive velocity vectors
+or three locations, if it can be measured at a point it can be 
+reported directly.  It is useful for projecting trajectories
+into the future.  Reported accelerations should subtract a 
+constant of 9.80665 m/s^2 from the reported values to account
+for the effect of gravity, independent of the actual value of
+gravity at the location.  For applications that need additional
+precision, this known constant can be added back in and a
+more accurate estimate of gravity at that location subtracted
+instead.
+
+#### Time
+For dynamic data, a timestamp should be provided along with the 
+data so that the time of last update is known.  In combination
+with motion information (velocity, acceleration) this can be used
+to project uncertainty into the future.
+
+#### Feature of Interest
+The location of a sensor and the location of the thing being
+measured may not be the same.  If only one location is given they
+are assumed to be coincident.  However, for all values
+versions with "foi" prepended, e.g. "foiLocation" may be used
+in which case this is considered to be the location of the point
+being measured, and the corresponding "plain" value,
+e.g. "location", is the location of the sensor.
+
+#### Accuracy
+Clear about radius vs. interval e.g. for lat/long.
+Probability of inclusion in region.
+Alternatives variance, prob distribution (eg. Gaussian).
+Can apply accuracy to any of the above measurements.
+
+#### Units
+Need to be able to specify units but should also have clear defaults.
+
+#### Logical Regions and Relationships
+Sometimes is it more convenient to refer to a labelled region or zone
+than a measured location.  These "logical" locations can be used instead
+of or in addition to numerical locations.
+Example of logical locations include:
+* Named rooms in a building
+* Floor in building
+* Plots in a city
+* Roads
+It may also be useful to represent
```

Comment:
Maybe another important use of OWL ?

Reply:
Yes, and there are ontologies already defined for this that we should track down and reuse.